activity_log.php

```diff
@@ -30,9 +30,6 @@
 $sys = ( $is_admin && getGetValue ( 'system' ) != '' );
 
 print_header();
-
-ob_start();
```

Comment:
Can you recall why all these ob_start calls are in the code?  It seems unnecessary, but they must have been put there for a reason(?).